server.js

```diff
@@ -26,12 +26,24 @@ if (!isProduction) {
     },
   });
 
+  function setPanoptesEnvMiddleware(req, res, next) {
+    if (req.url === '/' && !req.url.match(/\W?env=(\w+)/) {
```

Comment:
Am I reading this right? If `req.url === '/'`, then `req.url` will never `.match(/\W?env=(\w+)/)`.

And is it missing a closing `)`?